packages/kernel-5.10/mkfs.xfs.conf

```diff
@@ -0,0 +1,5 @@
+# V5 features that were the mkfs defaults when the upstream Linux 5.10 LTS
+# kernel was released at the end of 2020.
+
+[inode]
+nrext64=0
```

Comment:
Yea, this is indeed the only feature we care about to fix the original issue. Unless you have more thoughts and we can sure add more in other PRs -- I still think we should only keep this one here because this is the one solving the problem.